src/Voron/Impl/LowLevelTransaction.cs

```diff
@@ -1377,12 +1377,13 @@ public void Rollback()
             if (_txState.HasFlag(TxState.Disposed))
                 ThrowObjectDisposed();
 
-
             if (Committed || RolledBack || Flags != (TransactionFlags.ReadWrite))
                 return;
 
             OnRollBack?.Invoke(this);
 
+            _freeSpaceHandling.OnRollback();
```

Comment:
Shouldn't it be `_freeSpaceHandling.OnRollback?.Invoke()` to prevent potential NRE?

Reply:
`OnRollback` is a function and cannot be `null`.
https://github.com/ravendb/ravendb/pull/19429/files#diff-c4cbc0f7b25bab56b6502ea80de103fb97842a178ef8d6d132495c5c14689a4bR16